app/views/users/new.html.erb

```diff
@@ -0,0 +1,56 @@
+<!--
+<form  method="POST" action="/users" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label>Name:</label>
```

Comment:
You should also add the `for` attribute for label tag ex. `<label for='user[username]'>Username</label>`